identity/src/commonMain/kotlin/com/android/identity/nfc/CommandApdu.kt

```diff
@@ -48,31 +47,28 @@ data class CommandApdu(
         if (payload.size > 0) {
             lcPresent = true
             if (useShortEncoding) {
-                bsb.append(payload.size.toByte())
+                bsb.appendUInt8(payload.size)
             } else {
-                bsb.append(0x00)
-                bsb.append((payload.size shr 8).toByte())
-                bsb.append(payload.size.toByte())
+                bsb.appendUInt8(0u) // TODO: b/393388370 looks strange if the goal is to just add Int16.
+                bsb.appendUInt16(payload.size)
             }
             bsb.append(payload)
         }
         if (le > 0) {
             if (useShortEncoding) {
                 if (le == 0x100) {
-                    bsb.append(0x00)
+                    bsb.appendUInt8(0)
```

Comment:
nit: we're inconsistent about whether this is a signed 0 or an unsigned one... It doesn't really matter, but we may want to stick with one, just so the reader doesn't wonder why we use a signed one in one place and an unsigned one elsewhere.